x/gov/keeper/keeper.go

```diff
@@ -193,22 +193,22 @@ func (k Keeper) ModuleAccountAddress() sdk.AccAddress {
 // validateProposalLengths checks message metadata, summary and title
 // to have the expected length otherwise returns an error.
 func (k Keeper) validateProposalLengths(metadata, title, summary string) error {
-    if err := k.assertMetadataLength(metadata); err != nil {
```

Comment:
How come the spacing was wrong?
I believe we have gofmt as linter.
I'll need to check when reviewing the linters config. Nothing for you to do here, just noting.